src/browser-frontend/views/browser/chrome/tabbar/tab.css

```diff
@@ -47,6 +48,32 @@ specific language governing permissions and limitations under the License.
   height: var(--theme-tab-open-height);
 }
 
+.tab.has-owner {
+  left: 0;
+  transition:
+    width 0.05s var(--animation-curve),
+    background-position 0.2s var(--animation-curve),
+    height 0.2s var(--animation-curve),
+    left 0.2s var(--animation-curve);
+}
+
+.tab.has-owner * {
+  opacity: var(--theme-tab-open-opacity);
+  transform: translate(0, 0);
+  transition: opacity 0.2s 0.2s var(--animation-curve), transform 0.2s 0.2s var(--animation-curve);
+}
+
+.tab.has-owner.initial * {
+  opacity: var(--theme-tab-initial-opacity);
+  transform: translate(0, 30px);
```

Comment:
This 30px should be a variable as well, probably `--theme-tabbar-height` minus the `.tabbar`'s top padding (which should probably also turn into a variable)? 

Alternatively, we could have a `--theme-tab-height` instead of `--theme-tabbar-height`, which would indirectly determine the `.tabbar`'s height based on on `.tab` heights, but this change seems more involved with little benefit.